ee/clickhouse/models/person.py

```diff
@@ -75,7 +75,7 @@ def create_person(
 
 
 def create_person_distinct_id(team_id: int, distinct_id: str, person_id: str, sign=1) -> None:
-    data = {"distinct_id": distinct_id, "person_id": person_id, "team_id": team_id, "sign": sign}
+    data = {"distinct_id": distinct_id, "person_id": person_id, "team_id": team_id, "_sign": sign}
```

Comment:
hmm, I don't get how this would've borked things.

The parameter can be called "sign", and as long as it's position is correct, and the data is correct, it would be fine?

Reply:
The kafka message is unrelated to that query being passed in. I think the way the kafka message is defined it requires that the parameters need to match exactly @fuziontech?

Reply:
Yup! Exactly. The Kafka message must match the kafka engine backed table exactly.
https://github.com/PostHog/posthog/tree/master/ee/clickhouse#L135 which in this case is `_sign` so any payloads going to kafka from this function have been missing the `_sign` column.